### PR TITLE
 Merge v4.2.2 from 'latest' into  'release-v5.0.0'

### PR DESCRIPTION
**Pull request by jemrobinson**
## :white_check_mark: Checklist

- [x] You have given your pull request a meaningful title (_e.g._ `Enable foobar integration` rather than `515 foobar`).
- [x] You are targeting the appropriate branch. If you're not certain which one this is, it should be **`develop`**.
- [x] Your branch is up-to-date with the **target branch** (it probably was when you started, but it may have changed since then).

### :vertical_traffic_light: Depends on

n/a

### :arrow_heading_up: Summary

Merge changes to ensure that `develop` can be cleanly merged into `latest` when required.

### :closed_umbrella: Related issues

n/a

### :microscope: Tests

n/a

### CODE REVIEW

**Comment by JimMadge** [comment]
I changed the target branch to the release branch.

**Comment by jemrobinson** [comment]
@JimMadge : sure - although really this should have been done into `develop` immediately after we released `v4.2.2`.

**Comment by JimMadge** [comment]
Probably. I think the branches have diverged enough now that the merge will probably be mostly to discard changes from 4.2.2.

**Comment by jemrobinson** [comment]
Yes, it is - although with changes to docs it was worth manually checking that the same changes had made it into the v5 docs too.

File: .lychee.toml [hunk 1]
```diff
@@ -97,6 +97,7 @@ exclude = [
   'opensource\.org',  # 403
   'portal\.azure\.com',  # 403
   'turing\.ac\.uk',  # DDOS protection
+  'web\.archive\.org', # DDOS protection
```
Comment (JimMadge) [inline .lychee.toml]: Do we have any links to this domain?

I think it was added for docs which had been removed from the Azure site.

Comment (jemrobinson) [reply .lychee.toml]: Ah OK. I'll remove it then.

**Comment by jemrobinson** [comment]
39 commits to make 0 changes to files :)

**Pull request merged**